examples/supertuxkart/fleet.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usually you would define a Fleet rather than a GameServerSet
```

Comment:
Does this comment still apply? It seems like it is defining a fleet.

Reply:
Nope, thanks for catching that. Looks like that's a long heldover copy paste in comment that got moved to various places. Removed!